docs/feature-toggles.rst

```diff
@@ -77,3 +77,116 @@ and documented in the admin site's `constance section`_.
 .. _Waffle switches: https://waffle.readthedocs.io/en/latest/types/switch.html
 .. _Waffle flags: https://waffle.readthedocs.io/en/latest/types/flag.html
 .. _PR 3331: https://github.com/mozilla/kuma/pull/3331
+
+Using Traffic Cop for A/B Testing
+==================================
+
+Traffic Cop is a lightweight JavaScript library that allows conducting content experiments on MDN without the need for third party tools such as Optimizely. Traffic Cop also respects user privacy, and will not run if Do Not Track is enabled.
+
+On MDN, Traffic Cop will also not run for logged in users. Below follows the steps a developer needs to follow to implement a Traffic Cop content experiment.
+
+The first task is to choose, or create, your control, and test pages. We will here use a scenario of testing a new feature on MDN. The new feature involved replacing the static examples on top, with an interactive editor.
+
+First we choose our control page. For this example, we will use:
+
+    https://developer.mozilla.com/docs/Web/JavaScript/Reference/Global_Objects/Array/push
+
+Next we need our test page. The convention on MDN is to first create a base page that follows a naming convention such as:
+
+    https://developer.mozilla.org/docs/Experiment:ExperimentName/
+
+With the base page created, we next create out test variation of our chosen page. To do this, navigate to:
+
+    https://developer.mozilla.org/docs/Experiment:ExperimentName/Array.prototype.push()
+
+You should should now be presented with an editing interface to create the new page.
+
+Switch back to the control page, and click the ``Edit`` button. Switch the editor to source mode and copy all of the contents inside the editor. Switch back to our test, ensure the editor is set to source mode, and paste the content you just copied.
+
+You can now go ahead and make the changes you wish to user test. Once you are complete, go ahead and puclish the page.
+
+The JavaScript
+==============
+
+Let us first add the JavaScript code to initialise and configure Traffic Cop.
+
+Inisde ``kuma\static\js``, create a new file called some like ``experiment-experiment-name.js`` and paste the following code into the file:
```

Comment:
s/Inisde/Inside/

There should be a double-colon after ``file``, like ``file::``, to tell Sphinx that the following is [source code](http://www.sphinx-doc.org/en/stable/rest.html#source-code) instead of just a block quote.

Reply:
👍